### ISSUE TITLE
Getting issue in using the setup.

### ISSUE THREAD

**Issue by prashantidealittechno** [graphaware/reco4php#12]
Can you please help me how can I solve this issue: 

On example.php with sample given in README FILE.

```
Fatal error: Declaration of GraphAware\Reco4PHP\Demo\Github\FollowedByFollowers::buildScore(GraphAware\Common\Type\NodeInterface $input, GraphAware\Common\Type\NodeInterface $item, GraphAware\Common\Result\RecordViewInterface $record, GraphAware\Reco4PHP\Context\SimpleContext $simpleContext) 
must be compatible with 
GraphAware\Reco4PHP\Engine\DiscoveryEngine::buildScore(GraphAware\Common\Type\Node $input, GraphAware\Common\Type\Node $item, GraphAware\Common\Result\Record $record, GraphAware\Reco4PHP\Context\Context $context): GraphAware\Reco4PHP\Result\SingleScore in /var/www/html/reco_2/_demo/github/FollowedByFollowers.php 
on line 11
```

**Comment by prashantidealittechno** [comment]
Hello I made changes to all the functions it shows me the same error and now their is no error but also no result, 

I thinks it is because of database not matched with the query written on file */_demo/github/FollowedByFollowers.php*

**When i print I get the below code:**
```
GraphAware\Common\Cypher\Statement Object
(
    [text:protected] => MATCH (input) WHERE id(input) = {id}
        MATCH (input)<-[:FOLLOWS]-(follower)-[:FOLLOWS]->(reco)
        WHERE size((follower)-[:FOLLOWS]->()) < {max_follows}
        RETURN reco, count(*) as score
        LIMIT 100
    [parameters:protected] => Array
        (
            [id] => 34228
            [max_follows] => 200
        )

    [tag:protected] => 
    [type:protected] => GraphAware\Common\Cypher\StatementType Object
        (
            [value:protected] => STATEMENT_READ_WRITE
        )

)
```
Can anybody help me how to fix this query to make it compatible with  **MovieLens Latest Datasets (ml-latest-small.zip)**

**Comment by ikwattro** [comment]
Hi @prashantidealittechno 

Sorry for delay I was on holidays and am now right up at LaraconEU. I will try to check this tonight.

**Comment by prashantidealittechno** [comment]
Hey ikwattro, no worry, I checked and found that it is not the issue as the setup I am trying  to run is with example.php but I then found that You add it in your README file to run the setup with MovieLens Latest Database. I run the setup successfully in my local environment. 

I got some issue because of using php7 function return type. but by passing the type in front of function I got the result. 
![movie_recommendation](https://cloud.githubusercontent.com/assets/8256764/17926618/6fc9bfdc-6a0f-11e6-80f5-cf9ee9d30542.png)
